server/services/reportsService.ts

```diff
@@ -30,9 +38,9 @@ export default class ReportsService {
     )
   }
 
-  private static isUnallocated(pd: OffenderDetail) {
-    const results = pd.offenderManagers.some(
-      (om) => om.active === true && (om.staff.unallocated === true || om.staff.code.endsWith('U'))
+  private static comAllocated(pd: OffenderDetail) {
```

Comment:
edited to check for com allocated rather than not allocated to be easier to read